packages/app/src-tauri/src/commands/common.rs

```diff
@@ -0,0 +1,56 @@
+use derive_new::new;
+use getset::Getters;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Getters, new)]
+pub struct EndPointPayloadConf {
+    #[getset(get = "pub ")]
+    file_name: String,
+
+    #[getset(get = "pub ")]
+    path: Option<String>,
+
+    http_host: String,
+    https: bool,
+}
+
+#[derive(Debug, Getters, PartialEq, Serialize)]
```

Comment:
is there a reason why we are creating the getters? can we access directly using dot props ex `info.url`

Reply:
Used to protect the prop